projects/core/src/i18n/i18next/i18next-init.ts

```diff
@@ -50,3 +56,23 @@ export function syncI18nextWithSiteContext(language: LanguageService) {
   // always update language of i18next on site context (language) change
   language.getActive().subscribe(lang => i18next.changeLanguage(lang));
 }
+
+/**
+ * Returns a function appropriate for i18next to make http calls for JSON files.
+ * See docs for `i18next-xhr-backend`: https://github.com/i18next/i18next-xhr-backend#backend-options
+ *
+ * It uses Angular HttpClient under the hood, so it works in SSR.
+ * @param httpClient Angular http client
+ */
+export function i18nextGetHttpClient(
```

Comment:
can we cover it with some basic test, just for sake of code coverage or against unintentional break by someone?